### PR TITLE
Removed click listener from card icon

### PR DESCRIPTION
**Pull request by Biren-Nayak**
#449 
-  After removing the additional click listener in card icon, clicking the card icon itself works just like clicking in the rest of the card.

<table><tr>
<td>
</td>
<td>
</td></tr>
</table>

### CODE REVIEW

File: app/src/main/java/protect/card_locker/LoyaltyCardCursorAdapter.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ public void onBindViewHolder(LoyaltyCardListItemViewHolder inputHolder, Cursor i
 
     private void applyClickEvents(LoyaltyCardListItemViewHolder inputHolder, final int inputPosition)
     {
-        inputHolder.mThumbnailContainer.setOnClickListener(inputView -> mListener.onIconClicked(inputPosition));
+        //inputHolder.mThumbnailContainer.setOnClickListener(inputView -> mListener.onIconClicked(inputPosition));
```
Comment (TheLastProject) [inline app/src/main/java/protect/card_locker/LoyaltyCardCursorAdapter.java:131]: Please don't just comment out code but actually remove it.

**Review by TheLastProject — CHANGES_REQUESTED** [review]
Please remove the code instead of just commenting it out.

Also, because this is the only place using onIconClicked, please also remove the function itself (doing a quick search for onIconClicked should show you the places it's defined)

**Comment by Biren-Nayak** [comment]
@TheLastProject done

**Comment by TheLastProject** [comment]
Seems perfect now, thanks :)

**Pull request merged**